packages/conformance/src/commands/verify.ts

```diff
@@ -42,6 +45,18 @@ export default class Verify extends Command {
   public async run(): Promise<void> {
     const { args, flags } = await this.parse(Verify);
 
+    const trustedRootPath = flags['trusted-root'];
+    const trustMaterial = trustedRootPath
+      ? await trustMaterialFromPath(trustedRootPath)
+      : await trustMaterialFromTUF(flags['staging']);
+
+    const verifier = new Verifier(trustMaterial, {
+      tlogThreshold: 0,
+      tsaThreshold: 0,
+    });
```

Comment:
To use a custom trusted-root, we need to instantiate the `Verifier` directly instead of relying on the default verify function implemented in the `client` package.